src/sticker/generator.py

```diff
@@ -43,7 +43,8 @@ async def generate_image(self, achievement_text: str) -> Image:
         prompt = (
             f'Create a humorous sticker representing the achievement of {achievement_text_in_en}. '
             f'The design should be in the form of a vector graphic with simple lines and a limited '
-            f'color palette. The background should be white to emphasize the sticker format')
+            f'color palette. The background should be white to emphasize the sticker format.'
+            f'Make sticker circular.')
```

Comment:
What do you think about phrasing it this way? +`framed` keyword, and no additional sentence
```suggestion
            f'Create a circular framed humorous sticker representing the achievement of '
            f'{achievement_text_in_en}. The design should be in the form of a vector graphic '
            f'with simple lines and a limited color palette. The background should be white to '
            f'emphasize the sticker format')
```